lib/app/components/pages/profile/controllers/profile_controller.dart

```diff
@@ -4,8 +4,7 @@ class ProfileController extends GetxController {
   //TODO: Implement ProfileController
 
   final count = 0.obs;
-
-
+  final userName = "".obs;
```

Comment:
```suggestion
  ProfileController({required this.userName});
  final String userName;
```
②上記のように修正お願いします。
③あとProfileBindingも以下のように修正お願いします。
```
class ProfileBinding extends Bindings {
  @override
  void dependencies() {
    Get.lazyPut<ProfileController>(
      () => ProfileController(userName: Get.arguments[0]),
    );
  }
}
```